client/errors.go

```diff
@@ -32,23 +32,16 @@ const (
 // This list is incomplete and undefined errors may also be returned.
 // Errors returned from this package may be tested against these errors with errors.Is.
 var (
```

Comment:
- Yes, all deleted errors were unused. It's because the linter doesn't catch unused public variables when they are become unused by developers, the same way it catches unused private variables.

Reply:
Okay, thanks then for deleting them :)